letsencrypt/client.py

```diff
@@ -117,6 +120,12 @@ class Client(object):
 
     """
 
+    # Message to show if enabling a redirect fails, but recovery is successful
+    _SUCCESSFUL_REDIRECT_RECOVERY_MSG = (
+        "We were unable to setup a redirect for your server, however, we "
+        "successfully installed your certificate. If you'd like to revert "
+        "these changes as well, run 'letsencrypt rollback'.")
```

Comment:
This is confusing language.  Does the user know what other reversion is referred to by the words "as well"?  Do they need to know?  I think probably we should just strike "~~If you'd like to revert these changes as well, run letsencrypt rollback~~", but if we want to keep it, find a narrative of when and why they would want to run rollback now.

"setup" -> "set up"?